src/main/java/domain/GameResult.java

```diff
@@ -0,0 +1,47 @@
+package domain;
+
+public enum GameResult {
+    WIN("승"),
+    LOSE("패"),
+    DRAW("무");
+
+    private final String title;
+
+    GameResult(String title) {
+        this.title = title;
+    }
+
+    public static GameResult from(Player player, Dealer dealer) {
+        if (player.isBust()) {
+            return LOSE;
+        }
+        if (dealer.isBust()) {
+            return WIN;
+        }
+        if (dealer.isBlackJack() && player.isBlackJack()) {
+            return DRAW;
+        }
+        if (dealer.isBlackJack()) {
+            return LOSE;
+        }
+        if (player.isBlackJack()) {
+            return WIN;
+        }
+
+        return getResultByTotalScore(player, dealer);
+    }
```

Comment:
```suggestion
    public static GameResult from(Player player, Dealer dealer) {
        if (hasBust(player, dealer)) {
            return getBustCaseResult(player);
        }
        if (hasBlackJack(player, dealer)) {
            return getBlackJackCaseResult(player, dealer);
        }
        return getResultByTotalScore(player, dealer);
    }

    public static boolean hasBust(Player player, Dealer dealer) {
        return player.isBust() || dealer.isBust();
    }

    public static GameResult getBustCaseResult(Player player) {
        if (player.isBust()) {
            return LOSE;
        }
        return WIN;
    }

    public static boolean hasBlackJack(Player player, Dealer dealer) {
        return player.isBlackJack() || dealer.isBlackJack();
    }

    public static GameResult getBlackJackCaseResult(Player player, Dealer dealer) {
        if (player.isBlackJack() && dealer.isBlackJack()) {
            return DRAW;
        }
        if (player.isBlackJack()) {
            return WIN;
        }
        return LOSE;
    } 
```
개인적 취향인데 이렇게 메서드 추출해 볼 수 있을 것 같네요~